scrapers/FemdomEmpire.yml

```diff
@@ -0,0 +1,63 @@
+name: "FemdomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+      - feminized.com/tour/trailers/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/models/
+      - feminized.com/tour/models/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3
+      Date:
+        selector: //p[span[contains(.,"Date Added")]]
+        postProcess:
+          - replace:
+            - regex: "Date Added:"
+              with:
+          - parseDate: January 2, 2006
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]/a
+      Tags:
+        Name: //ul/li[@class="label" and contains(.,"Categories:")]/..//a
+      Studio:
+        Name:
+          selector: //div[@id='logo']/h1/a/span
+        Image:
+          selector: //img[@class="update_thumb thumbs stdimage"]/@src0_1x
+          postProcess:
+            - replace:
+              - regex: .*/tour/content//
+                with: "https://feminized.com/tour/content/"
+  performerScraper:
+    common:
+      $performerInfo: "//div[@class='contentBox'][1]"
+      # if stash scraper supports xpath 2 we might be able to parse measurements etc from femdomempire stats lists.
+      # $profileAbout: "$performerInfo/div[@class='profile-about']/p"
+    performer:
+      Name:
+        selector: "$performerInfo/div[@class='profile-details']/h3"
+      Gender:
+        selector: "$performerInfo/div[@class='title']/h3"
+        postProcess:
+          - replace:
+            - regex: "^Mistress Sissy .*$"
+              with: Male
+          - replace:
+            - regex: "^Mistress (.*)$"
+              with: Female
+      Image:
+        selector: "//li[@class='flex-active-slide']/img/@src"
+      URL:
+        selector: "//link[@rel='canonical']/@href"
```

Comment:
there is no canonical link in the page so it doesnt do anything

Reply:
Yeah. I was under the impression from some other piece of documentation that the `URL` has to exist - this is my first time adding a scraper so I wasn't sure where to look for the ByURL-type scrapers. I'll remove it :-)